easybuild/tools/filetools.py

```diff
@@ -180,10 +180,13 @@ def resolve_path(path):
 
     :param path: path that (maybe) contains symlinks
     """
-    try:
-        resolved_path = os.path.realpath(path)
-    except OSError as err:
-        raise EasyBuildError("Resolving path %s failed: %s", path, err)
+    if path is None:
```

Comment:
@wpoely86 Well, sure, that's another option, but then we're bound to make the same mistake again later?

It would boil down to doing this in `get_paths_for`:

```python
    # figure out installation prefix, e.g. distutils install path for easyconfigs
    eb_path = which('eb')
    if eb_path is None:
        _log.warning("'eb' not found in $PATH, failed to determine installation prefix")
    else:
        # real location of eb should reside in <install_prefix>/bin/eb
        eb_path = resolve_path(eb_path)
        install_prefix = os.path.dirname(os.path.dirname(eb_path))
```

whereas now we have:

```python
    # figure out installation prefix, e.g. distutils install path for easyconfigs
    eb_path = resolve_path(which('eb'))
    if eb_path is None:
        ...
```

I'm not sure which one is better, I'm happy to change it as you suggest if you feel strongly about it.